presto-main/src/main/java/io/prestosql/memory/LocalMemoryManager.java

```diff
@@ -38,6 +39,7 @@
 {
     public static final MemoryPoolId GENERAL_POOL = new MemoryPoolId("general");
     public static final MemoryPoolId RESERVED_POOL = new MemoryPoolId("reserved");
+    private static final int AVAILABLE_PROCESSORS = ManagementFactory.getOperatingSystemMXBean().getAvailableProcessors();
```

Comment:
This probably shouldn't be a constant since it can technically change at runtime

Also static import `getOperatingSystemMXBean`